### ISSUE TITLE
Blocks mode code examples missing ARIA name

### ISSUE THREAD

**Issue by daminihs** [wordplaydev/wordplay#690]
### What happened?

WCAG Rule: Ensure every ARIA input field has an accessible name.

1. On the Guide page -> how to -> create and control characters
All 6 boxes do not have an accessible name. I'm not sure why these are categorized as input fields. Once you click on any of them, the code is not editable and the editor doesn't open so not sure if the behavior is correct. 
![Image](https://github.com/user-attachments/assets/a88480aa-e874-4bc3-b01d-048b568c4ae5)

2. Same problem for all keyword elements too. Guide page -> concepts 

![Image](https://github.com/user-attachments/assets/241561da-6448-4ae6-a898-d7f407c3df3f)

To solve this problem, you need to fix at least (1) of the following:
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute

https://dequeuniversity.com/rules/axe/4.10/aria-input-field-name?application=AxeChrome

### What browsers are you seeing the problem on?

Chrome

### What operating system are you using?

Windows

**Comment by amyjko** [comment]
Ah, yes, these are inactive text fields because the editor is in blocks mode. Hm, I'll think about this one.

**Comment by daminihs** [comment]
This issue still exists for every code block on the Guide How to page

![Image](https://github.com/user-attachments/assets/891e103e-86f3-4afb-af9d-2b611ee04d44)

**Comment by Yabtse** [comment]
How could I get started on this

**Comment by amyjko** [comment]
Assigned! Sorry for the delay on this one. (@Yabtse, remember to tag me, per the dev wiki guidelines).

**Comment by amyjko** [comment]
* [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
* Blocks mode is not active, so to activate it...
* `NodeView.svelte` is the base component for the blocks mode editor
* Make this line in `RootView` `true` instead of `blocks`: `let isBlocks = writable<boolean>(blocks);`

**Comment by amyjko** [comment]
Are you still working on this @Yabtse?

**Comment by amyjko** [comment]
@Yabtse What's the status of this? I'm working on reimplementing the blocks editing mode, so it'd be helpful to know if you're working on this.

**Comment by Yabtse** [comment]
@amyjko, I apologize. I ended up diving too deep into issue #770, and I don’t think I’ll be able to finish this task.

**Comment by amyjko** [comment]
Thanks for the update. I'm going to close this; I can no longer reproduce it, so I believe it is resolved in the `blocks` branch.
